.gitignore

```diff
@@ -0,0 +1,11 @@
+/target
+Cargo.lock
+
+# Editors
+
+# IntelliJ
+.idea
```

Comment:
Well, this part comes from the gitignore file of servo, so I guess it should be OK?